### ISSUE TITLE
What is the minimal supported NetStandard version?

### ISSUE THREAD

**Issue by damianh** [damianh/LibLog#150]
Fellow LibLog users,

I've _finally_ opened a PR to move to netstandard and .net core sdk #149  :)

A little delayed mostly because I personally held off .net core until version 2.0 and RTM bugs with content packages with code transforms.

My experience with NetStandard so far is that 
 1. Working with v < 2.0 libraries in .NET Framework (NET 4.5/4.6) projects has been pretty painful. Mostly a reference mess, sometimes something more sinister (complication errors, runtime foibles etc)
 2. Working with NetStandard2.0 libraries in net461 projects (and netcoreapp2.0) has been very smooth.

Basically, I don't see myself ever using `netstandard1.x` in any project or library.

However, to support netstandard1.x I have to follow the same path at supporting PCL previously. Specifically `Type.GetTypeInfo()` vs `Type.GetType()`. Not a huge deal, but I'd rather not do it if it's not needed. There are probably some other gotchas such as additional nuget packages the must be referenced.

Does anyone have any arguments why LibLog should support for netstandard < 2.0?

Thx in advance for sharing.

**Comment by snakefoot** [comment]
If anyone is making libraries that should be used by UWP-apps for Windows-devices without Fall Creators update, then they need the support.

**Comment by damianh** [comment]
I'm trying to parse that a bit... Are you saying that UWP-apps can only use `netstandard2.0` if they are **developed and deployed** on Win10 w/ Fall Creators update? Got a link for that?

**Comment by snakefoot** [comment]
> Got a link for that?

Yes UWP 1.0 (uap10.0) only works with NetStandard 1.4 (or older) -> https://docs.microsoft.com/en-us/dotnet/standard/net-standard

And only only a subset of the dependencies (Ex. no System.Threading.Thread), and reflection can throw funny exception because of NET Native funny business

**Comment by damianh** [comment]
Thanks. Looks like we should support. 

> reflection can throw funny exception because of NET Native funny business

Just wait until ILinker and CoreRT come along. I think people have gotten too comfortable with non-optimized stack traces in their log messages...

**Comment by damianh** [comment]
For LibLog 5, going to target netstandard2.0. Netstandard1.x just didn't get wide adoption and netstandard 2.x is supported on [all platforms](https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.0.md). People who really need netstandard support < 2.x can use the current copy-paste method with `LIBLOG_PORTABLE`.
